odoo/exceptions.py

```diff
@@ -12,13 +9,16 @@
     check out the :mod:`odoo.addons.test_exceptions` module.
 """
 
+from http import HTTPStatus
```

Comment:
I actually don't like the fact that now exceptions depend on http, event more when it's just for a few constants.